backend/src/services/secret-v2-bridge/secret-v2-bridge-dal.ts

```diff
@@ -359,7 +353,9 @@ export const secretV2BridgeDALFactory = (db: TDbClient) => {
         .whereIn("folderId", folderIds)
         .where((bd) => {
           if (filters?.search) {
-            void bd.whereILike("key", `%${filters?.search}%`);
+            if (filters?.includeTagsInSearch)
```

Comment:
Two things here: 

1. Please put a brackets when its if with else. Because reading would be quite hard to catch. If its not with else no need for brackets.
2. Also as there are this many joins in place for this query. Let's specifically add the table names like
```
`${TableName.SecretV2}.key}
```

This is avoid later causing issue due to ambiguous key when table changes.

Reply:
add table names to query and brackets to `if` statement